docs/targets/salesforce.md

```diff
@@ -0,0 +1,168 @@
+# Event Target for Salesforce
+
+This event Target receives [CloudEvents][ce] and invokes a Salesforce endpoint.
+
+## Prerequisites
+
+### Salesforce Account
+
+Salesforce target uses [OAuth JWT credentials][salesforce-oauth-jwt] for service authentication.
+
+1. First, you will need to generate an X509 certificate for signing and verifying requests.
+We will be using `OpenSSL` but any other certificate generation tool should work.
+
+```sh
+openssl req -x509 -sha256 -nodes -days 36500 -newkey rsa:2048 -keyout tm-sf.key -out tm-sf.crt
+```
+
+2. At Salesforce site select `Setup > Apps > App Manager`, click on `New Connected App`.
+
+    - Fill in mandatory fields, then click `Enable OAuth Settings`.
+    - A callback URL is mandatory but can be filled with any HTTPS data.
+    - Enable `Use digital signatures` and upload the public cert (`tm-sf.crt` in the example above).
+    - Add Scopes for `api` and `refresh_token, offline_access`.
+    - Save.
+
+    ![Salesforce connected app](../images/salesforce/salesforce-connected-app.png)
+
+    - Select the Connected App from the list and at the click on `Manage`.
+    - Click `Edit policies`.
+    - Set Permitted users to `Admin approved users are pre-authorized`.
+    - Save.
+
+    ![Connected app policies](../images/salesforce/connected-app-policies.png)
+
+    - Select the Connected App from the list and at the click on `Manage`.
+    - Click `Manage Profiles`.
+    - Add permissions on the data this user will have access to.
+    - Save.
+
+3. Retrieve OAuth data to configure Triggermesh Target.
+
+   - Select the Connected App from the list and at the click on `View`.
+   - Copy `Consumer Key`
+   - Reveal and copy `Consumer Secret`
```

Comment:
Same for at the top, this is showing as `5 Reveal and copy Consumer Secret`

Reply:
https://github.com/triggermesh/docs/pull/107/commits/101eb49184fa2455621b7d04200c5c1deadde984